app/jobs/kit_import_job.rb

```diff
@@ -9,25 +9,26 @@ class KitImportJob < ApplicationJob
 
   rescue_from(StandardError) do |e|
     logger.info "An error ocurred: #{e.message}"
+    logger.debug e.backtrace.join("\n")
   end
 
-  def perform(file:, logger:, user_id: nil)
+  def perform(file_or_folder, logger:, user_id: nil)
     @current_user = user_id ? User.find(user_id) : User.first
-    @file = file
     @logger = logger
     @project = nil
     @report_templates_dir = Configuration.paths_templates_reports
-    @temporary_dir = Dir.mktmpdir
+    @working_dir = Dir.mktmpdir
     @word_rtp = nil
 
-    FileUtils.cp file, temporary_dir
-    unzip
+    copy_kit_to_working_dir(file_or_folder)
+
     import_methodology_templates
     import_note_templates
     import_plugin_templates
     import_project_package
     import_project_templates
     import_report_template_files
```

Comment:
I'm getting some errors around this area since the kit included here does not have a project template and an RTP. I think we should add a guard to check for the existence of the relevant directories.